ts/mcl.ts

```diff
@@ -14,8 +14,8 @@ export type MessagePoint = mclG1;
 export type Signature = mclG1;
 export type PublicKey = mclG2;
 
-export type solG1 = [string, string];
-export type solG2 = [string, string, string, string];
+export type solG1 = [BigNumberish, BigNumberish];
+export type solG2 = [BigNumberish, BigNumberish, BigNumberish, BigNumberish];
```

Comment:
Based on my understanding of `mcl` and how these types are used in TS/JS code and at the contract level, this seems like a more appropriate type.